lib/msal-react/src/MsalProvider.tsx

```diff
@@ -77,7 +77,7 @@ export function MsalProvider({instance, children}: MsalProviderProps): React.Rea
     };
 
     return (
-        <MsalContext.Provider value={contextValue}>
+        <MsalContext.Provider value={ contextValue }>
```

Comment:
Hmm, admittedly not the biggest of this.

Reply:
Fair enough, do you prefer no spaces or no rule?

Reply:
I think I like this?

`{ "when": "never", "attributes": { "allowMultiline": false }, "children": true }`

Reply:
Done